### DISCUSSION TITLE
M2ePRO compatibility status

### DISCUSSION THREAD

**Discussion by empiricompany** [OpenMage/magento-lts#3694] [Questions & Answers] [answered]
Hi!
I have a new project underway that would require integrating m2epro for integration with Amazon. 

Can anyone give me some feedback if everything works with the latest version of OpenMage 20.2.0 and PHP8? Thank you!

**Comment by fballiano** [comment]
I use m2epro for a customer and it works fine with php8 and OM20.2, I had a bug for which I filed https://github.com/m2epro/magento1-extension/pull/20 but they fixed it in a later version (after pinging them many times), for sure they don't look at the github and they don't care about contributions.

**Reply by empiricompany** [reply]
thanks @fballiano. yes I had read that and other tickets

**Comment by sreichel** [comment]
> for sure they don't look at the github and they don't care about contributions.

It seemed to work better to send patches via ticket system. At least you got a reply.

**Comment by empiricompany** [comment, marked as answer]
I'm running successfully on latest OpenMage 20.12.2 / M2ePRO 6.75.1
